wallet/subnet/primary/examples/add-primary-validator/main.go

```diff
@@ -57,9 +57,13 @@ func main() {
 	avaxAssetID := pWallet.AVAXAssetID()
 
 	addValidatorStartTime := time.Now()
+	shortNodeID, err := ids.ShortNodeIDFromNodeID(nodeID)
+	if err != nil {
+		panic(err)
```

Comment:
Can we use `log.Fatal` here like the other errors

Reply:
Done